pom.xml

```diff
@@ -169,6 +169,13 @@
         <version>2.14.0</version>
         <scope>test</scope>
       </dependency>
+
```

Comment:
Looks like a missing dependency on nexus-git-utils?

```
.groovy:[15,8] 18. ERROR in /Users/jcava/Documents/workspace/jenkins-nexus-platform-plugin/src/main/java/org/sonatype/nexus/ci/iq/RemoteRepositoryUrlFinder.groovy (at line 15)
[ERROR] 	import com.sonatype.nexus.git.utils.repository.RepositoryUrlFinderBuilder
[ERROR] 	       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[ERROR] Groovy:unable to resolve class com.sonatype.nexus.git.utils.repository.RepositoryUrlFinderBuilder
```

Reply:
Added this to get it to compile:
```
      <dependency>
        <groupId>com.sonatype.nexus</groupId>
        <artifactId>nexus-git-utils</artifactId>
        <version>${nexus.scm.client.version}</version>
      </dependency>
```

Reply:
Seems like something is not quite right:
```
ERROR: Build step failed with exception
groovy.lang.MissingMethodException: No signature of method: com.sonatype.nexus.api.zz.eu.scan() is applicable for argument types: (java.lang.String, com.sonatype.nexus.api.iq.ProprietaryConfig, java.util.Properties, java.util.Collections$UnmodifiableRandomAccessList, java.util.Collections$UnmodifiableRandomAccessList, java.io.File, hudson.EnvVars) values: [test-app, com.sonatype.nexus.api.iq.ProprietaryConfig@537c111a, ...]
Possible solutions: scan(java.lang.String, com.sonatype.nexus.api.iq.ProprietaryConfig, java.util.Properties, java.util.List, java.util.List, java.io.File), any(), scan(java.lang.String, com.sonatype.nexus.api.iq.ProprietaryConfig, java.util.Properties, java.util.List), wait(), find(), grep()
```

Reply:
This should be coming in transitively from the nexus Java Api. That being said, due to obfuscation it's included in the nexus Java Api jar file. Previously the classes weren't accessible here because obfuscation renamed them. That's why the PR on nexus Java Api changes the proguard config to make them available here. If your build can't find them it probably means you're building with the wrong version of nexus Java Api. It should be built with the Java Api from the 2339 branch.

Reply:
That second error also seems to indicate the nexus java api artifact isn't correct, the missing method is one of the methods I changed in the java api

Reply:
This worked by pulling the latest after the dep was updated.